assets/airports/kabq.json

```diff
@@ -0,0 +1,2170 @@
+                "ONM": ["DUMMM", "ONM"],
+                "SJN": [["WUTUP", "A160+"], "LAMSE", "SJN"]
+            },
+            "draw": [
+                ["JETOK", "BOSQE", "DUMMM", "ONM*"],
+                ["BOSQE", "WUTUP", "LAMSE*"],
+                ["LAMSE", "SJN*"]
+            ]
+        },
+        "DOOKK3": {
+            "icao": "DOOKK3",
+            "name": "Dook Three",
+            "rwy": {
+                "21": [],
+                "26": []
+            },
+            "body": [["JETOK", "A90-|S250-"], ["BIYEN", "A110+"], ["DOOKK", "A115+"]],
+            "exitPoints": {
+                "CME": ["DADDY", "LARLE", "TACOZ", "CME"],
+                "MOLVE": ["FATTT", "MOLVE"],
+                "TXO": ["FATTT", "MOLVE", "TXO"]
+            },
+            "draw": [
+                ["JETOK", "BIYEN", "DOOKK", "DADDY", "LARLE", "TACOZ", "CME*"],
+                ["DOOKK", "FATTT", "MOLVE*"],
+                ["MOLVE", "TXO*"]
+            ]
+        },
+        "FYSTA3": {
+            "icao": "FYSTA3",
+            "name": "Fiesta Three",
+            "rwy": {
+                "21": [["^_RWY0301.27DME", "A62+"]],
+                "26": [["^_RWY0803.22DME", "A75+"]]
+            },
+            "body": [["BLUUN", "A110+|S250-"], ["BURQI", "A120+|S250-"], ["FYSTA", "A140+"]],
+            "exitPoints": {
+                "FTI": ["MACHO", "FTI"],
+                "JLPNO": ["POYOH", "JLPNO"]
+            },
+            "draw": [
+                ["_RWY0803.22DME", "BLUUN"],
+                ["_RWY0301.27DME", "BLUUN", "BURQI", "FYSTA", "MACHO", "FTI*"],
+                ["FYSTA", "POYOH", "JLPNO*"]
+            ]
+        },
+        "GRZZZ3": {
+            "icao": "GRZZZ3",
+            "name": "Grass Three",
+            "rwy": {
+                "3": [],
+                "8": [["^TYILR", "S230-"]]
+            },
+            "body": ["ROWHO", "GRZZZ", "DNIDE"],
+            "exitPoints": {
+                "FTI": ["FTI"],
+                "TCC": ["JLPNO", "TCC"]
+            },
+            "draw": [
+                ["TYILR", "ROWHO", "GRZZZ", "DNIDE", "FTI*"],
+                ["DNIDE", "JLPNO", "TCC*"]
+            ]
+        },
+        "JEMEZ3": {
+            "icao": "JEMEZ3",
+            "name": "Jemes Three",
+            "rwy": {
+                "3": [["^_RWY2102.32DME", "A69+"]],
+                "8": [["^TYILR", "S230-"]]
+            },
+            "body": [["GRREN", "A120+|S250-"], "FIREY"],
+            "exitPoints": {
+                "JEMEZ": ["JEMEZ"]
+            },
+            "draw": [
+                ["TYILR", "GRREN", "FIREY"],
+                ["_RWY2102.32DME", "GRREN"],
+                ["JEMEZ"]
+            ]
+        },
+        "JETOK3": {
+            "icao": "JETOK3",
+            "name": "Jeetock Three",
+            "rwy": {
+                "3": []
+            },
+            "body": [["^_RWY2101.50DME", "A64+|S210-"], ["DSPOT", "A90-|S210-"], ["JETOK", "A100+"]],
```

Comment:
210 speed restriction should not be on the `_RWY2102.50DME` fix.
`DSPOT` should be `A90+|S210-`